### DISCUSSION TITLE
notes/sagemath/20240829-sagemath-0x00

### DISCUSSION THREAD

**Discussion by giscus** [kands-code/kands-code.github.io#83] [General]
# notes/sagemath/20240829-sagemath-0x00

找到属于自己的光

https://blogs.qarks.top/notes/sagemath/20240829-sagemath-0x00.html

**Comment by ghost** [comment]
写的比较完善。👍

**Reply by kands-code** [reply]
谢谢评价🙏，我这几篇笔记也只是记录我自己学习的一些内容，整体的内容也是按照 SageMath 的官方教程来的，所以有些地方可能看起来比较“奇怪”，如果有哪里不太对，还请帮忙指出，谢谢！

**Comment by ghost** [comment]
不过 Sagemath 也不是很“火”吧。没有 MATLAB 之类的闭源软件“火”。

**Reply by kands-code** [reply]
嘛，相对来说“火”了，起码比起之前基本没人知道要好，总归是往好的地方发展的
